> @xiajingchun 你好请问 ，流水线执行失败，是因为流水线本身问题还是测试用例没通过呢？

@cccxgit 之前workflow有些问题，现在好了。但是lint的format check没过，你再改改：
```
Run git diff -U0 --no-color c1d657d3811892b716e2bbb76f86690186129335 HEAD | /usr/share/clang/clang-format-10/clang-format-diff.py -p1 | tee /tmp/.clang-format-diff
  
--- src/kvstore/NebulaStore.cpp	(before formatting)
+++ src/kvstore/NebulaStore.cpp	(after formatting)
@@ -368,12 +368,8 @@
       cfFactory = options_.cffBuilder_->buildCfFactory(spaceId);
     }
     auto vIdLen = getSpaceVidLen(spaceId);
-    engine = std::make_unique<RocksEngine>(spaceId,
-                                           vIdLen,
-                                           dataPath,
-                                           walPath,
-                                           options_.mergeOp_,
-                                           cfFactory);
+    engine = std::make_unique<RocksEngine>(
+        spaceId, vIdLen, dataPath, walPath, options_.mergeOp_, cfFactory);
   } else {
     LOG(FATAL) << "Unknown engine type " << FLAGS_engine_type;
   }
Error: Process completed with exit code 1.
```

--- a/src/kvstore/NebulaStore.cpp
+++ b/src/kvstore/NebulaStore.cpp
@@ -369,7 +369,7 @@ std::unique_ptr<KVEngine> NebulaStore::createEngine(GraphSpaceID spaceId,
     }
     auto vIdLen = getSpaceVidLen(spaceId);
     engine = std::make_unique<RocksEngine>(
-      spaceId, vIdLen, dataPath, walPath, options_.mergeOp_, cfFactory);
+        spaceId, vIdLen, dataPath, walPath, options_.mergeOp_, cfFactory);
   } else {
     LOG(FATAL) << "Unknown engine type " << FLAGS_engine_type;
   }

--- a/src/kvstore/NebulaStore.cpp
+++ b/src/kvstore/NebulaStore.cpp
@@ -368,12 +368,8 @@ std::unique_ptr<KVEngine> NebulaStore::createEngine(GraphSpaceID spaceId,
       cfFactory = options_.cffBuilder_->buildCfFactory(spaceId);
     }
     auto vIdLen = getSpaceVidLen(spaceId);
-    engine = std::make_unique<RocksEngine>(spaceId,
-                                           vIdLen,
-                                           dataPath,
-                                           walPath,
-                                           options_.mergeOp_,
-                                           cfFactory);
+    engine = std::make_unique<RocksEngine>(
+      spaceId, vIdLen, dataPath, walPath, options_.mergeOp_, cfFactory);
   } else {
     LOG(FATAL) << "Unknown engine type " << FLAGS_engine_type;
   }

